samples/SampleApp/Program.cs

```diff
@@ -19,15 +28,27 @@ public Program()
             // providers may be added to an ILoggerFactory at any time, existing ILoggers are updated
 #if !ASPNETCORE50
             factory.AddNLog(new global::NLog.LogFactory());
+
+            factory.AddSerilog(new Serilog.LoggerConfiguration()
+                .Enrich.WithMachineName()
+                .Enrich.WithProcessId()
+                .Enrich.WithThreadId()
+                .MinimumLevel.Debug()
+                .WriteTo.RollingFile("file-{Date}.txt", outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} {Level}:{EventId} [{SourceContext}] {Message}{NewLine}{Exception}")
+                .WriteTo.Sink(new RollingFileSink("file-{Date}.json", new JsonFormatter(), null, null))
+                .WriteTo.Sink(new FileSink("dump.txt", new RawFormatter(), null)));
 #endif
-            factory.AddConsole();
-            factory.AddConsole((category, logLevel) => logLevel >= LogLevel.Critical && category.Equals(typeof(Program).FullName));
+            //factory.AddConsole();
```

Comment:
Why are these commented? Perhaps I'm confused about the scenario for this but I thought it was to enable structure log formatting to the built-in loggers, such as the Console logger?